raven/processes/wps_hindcast.py

```diff
@@ -61,6 +61,13 @@ def model(self, request):
         params = self.parse_tuple(request.inputs.pop(name)[0])
         model = get_model(name)(workdir=self.workdir)
         model.assign("params", params)
+     
+        
+        """ if we have a hotstart file"""
+        if "rvc" in request.inputs:
+            rvc=request.inputs.pop('rvc')[0].file
+            model.resume(rvc)
```

Comment:
@richardarsenault I've commented this bit and the test still works. I thought you had issues with this ?

Reply:
Yes, the test still passes, but the rvc is not being used. The results are different. If we comment it out, the model starts with an empty rvc file.

Reply:
Ok, so I'll add an assert to test the result and then try to trace back why the model is not initialized where it should.

Reply:
OK good. Some info/notes that I took:
Problem is in RavenProcess.initialize.
Base.py
rvc = self.get_config(request, ids=("rvc",))["rvc"]
rvc n'est pas dans la request ?! mais est dans self...

Reply:
Maintenant réglé. là j'ai des problèmes avec climatology_esp. Ça marchait de ton bord ? Avec quelle version de ravenpy?